joomla/joomla-cms
# [Sql azure] Normalize joomla install sample blog sql file

### Summary of Changes

Normalizes all sql azure joomla install file.
- always use `SET IDENTITY_INSERT [#__xxxx] ON/OFF;`
- `INSERT` to `INSERT INTO`
- Quote all table names and db field names
- Always use `'` to quote string values.
- Code style
### Testing Instructions

Mainly code review. (to many changes need to check code in raw) 
https://github.com/andrepereiradasilva/joomla-cms/blob/sqlazure-sample-blog/installation/sql/sqlazure/sample_blog.sql
If anyone could test an install 3.6.x would be great. Don't have sql azure to test,
### Documentation Changes Required

None.
